README.md

```diff
@@ -110,7 +110,7 @@ Note: a user is only allowed to access his/her own profile data, or the profile
 Response: User profile details, or Error
 
 
-__Endpoint__: `/me/:token`
+__Endpoint__: `/me/:userId?token=<token>`
```

Comment:
The /me/ endpoint should only require a token, and not a user ID, since the token should uniquely identify the user

Reply:
I added the userId bit as it enables us to revoke or change access tokens without it affecting the underlying user

Reply:
If the user has a valid token, we should be able to locate the API key in the DB to uniquely identify the user without the userId. If we revoke the token, then we won't locate the user, and won't return a result, which is expected since we don't want the user to be able to get profile information without a valid token. 
The benefit of having the /me/ endpoint instead of just the /profile/ endpoint is that the userId doesn't need to be transmitted, so you save that tiny amount of data.